x-pack/plugins/alerting/common/routes/rule/apis/create/schemas/v1.ts

```diff
@@ -69,16 +71,19 @@ export const actionAlertsFilterSchema = schema.object({
   ),
 });
 
-export const actionSchema = schema.object({
+export const defaultActionSchema = schema.object({
   uuid: schema.maybe(schema.string()),
   group: schema.string(),
   id: schema.string(),
   actionTypeId: schema.maybe(schema.string()),
   params: schema.recordOf(schema.string(), schema.maybe(schema.any()), { defaultValue: {} }),
   frequency: schema.maybe(actionFrequencySchema),
   alerts_filter: schema.maybe(actionAlertsFilterSchema),
+  type: schema.maybe(schema.literal(RuleActionTypes.DEFAULT)),
```

Comment:
The `maybe` is for BWC with actions created before the system actions. If the type is `undefined`, it is assumed to be a standard action, not a system action.